### DISCUSSION TITLE
text-[#1A1A1A] not saported  letest vartion

### DISCUSSION THREAD

**Discussion by dipankarviacon123** [tailwindlabs/tailwindcss#9851] [Help]
in WordPress them npx tailwindcss -i ./tailwind.css -o ./style.css --watch 

not working 

  "devDependencies": {
    "tailwindcss": "^3.2.4"
  },

![image](https://user-images.githubusercontent.com/57744756/202094563-028b3353-1ad3-43bb-a8d4-18d19f50792a.png)

**Comment by wongjn** [comment]
Consider checking that the `templates/front-page.php` file is covered by the Tailwind `content` file globs and also that there is no other CSS rules that may be overriding the Tailwind-generated rule for the text color.
